src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java

```diff
@@ -75,6 +83,10 @@ public String getAbortNoteText() {
         return this.abortNoteText == null ? "" : this.abortNoteText;
     }
 
+    public String getUnstableNoteText() {
+        return this.unstableNoteText == null ? "" : this.unstableNoteText;
```

Comment:
When ternary is used and we have binary operators it is recommended to use parenthesis.
Ex.
<code>
   return (this.unstableNoteText == null) ? "" : this.unstableNoteText;
</code>